docs/onnxruntime_dependencies.dot

```diff
@@ -0,0 +1,88 @@
+digraph "GG" {
```

Comment:
Should also add a link to this page from the main readme to make it more easily discoverable, perhaps under https://github.com/microsoft/onnxruntime#system-requirements ?

Reply:
Sure.